### DISCUSSION TITLE
How to deal with million rows of dataset

### DISCUSSION THREAD

**Discussion by mhamza6051** [AammarTufail/python_ka_chilla2023#86] [Q&A]
how to deal with million of rows dataset because i have a dataset with million of rows and my laptop taking too much time to any query execuation.

**Comment by AzamBukhari** [comment]
As mentioned in todays lecture as well. Don't go for million rows initially. 

To start with: load a small data set (upne million rows waley dataset main se shuru ki 500 -600 rows/instances cut ker k alag CSV mai save ker lain) . Test your code on small first . once executed , then try bigger dataset.

Also keep in mind, before going for big data sets, we need to estimate our  PC/laptop limitation as well as mentioned by baba Aammar saheb., So take it easy at thhe moment ;)

**Comment by AzamBukhari** [comment]
![Screenshot_2023-02-11-03-03-48-908_com linkedin android](https://user-images.githubusercontent.com/121688289/218235858-092a3a71-e15b-4fd3-a2b7-37a10e1c5ed1.jpg)
